cmd/main2.go

```diff
@@ -0,0 +1,16 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/pkg/errors"
+)
+
+func ABAB() error {
+	return errors.New("hogehoge")
+}
+
+func Hoge() {
+	err := ABAB()
+	fmt.Println(err)
+	err = ABAB()
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
SA4006: this value of `err` is never used (staticcheck)